ledger/complete/wal/checkpoint_v6_writer.go

```diff
@@ -705,6 +705,7 @@ func decodeSubtrieCount(encoded []byte) (uint16, error) {
 	return binary.BigEndian.Uint16(encoded), nil
 }
 
+// TODO repace with utils/io.CloseAndMergeError
```

Comment:
Making this change in this PR will create lots of noise, I will do it in a separate PR